CPAC/utils/build_data_config.py

```diff
@@ -1268,15 +1293,19 @@ def update_data_dct(file_path, file_template, data_dct=None, data_type="anat",
 
         data_dct[site_id][sub_id][ses_id]['brain_mask'] = file_path
 
-    elif data_type == "func":
-
+    elif 'func' in data_type:
```

Comment:
Why change https://github.com/FCP-INDI/C-PAC/blob/50f39a28a77351563076dcb9636139ceb247e6c7/CPAC/utils/build_data_config.py#L1251 https://github.com/FCP-INDI/C-PAC/blob/50f39a28a77351563076dcb9636139ceb247e6c7/CPAC/utils/build_data_config.py#L1271 to https://github.com/FCP-INDI/C-PAC/blob/a693b608dde98fe9ea28a6bfab5e365710e5c60d/CPAC/utils/build_data_config.py#L1276 https://github.com/FCP-INDI/C-PAC/blob/a693b608dde98fe9ea28a6bfab5e365710e5c60d/CPAC/utils/build_data_config.py#L1296 ? It's fine, just curious

Reply:
i just like how that looks more than "==", but can change back

Reply:
no need, like I said, just curious